handler.go

```diff
@@ -44,6 +53,9 @@ func validateArguments(handler reflect.Type) (bool, error) {
 	return handlerTakesContext, nil
 }
 
+// validateReturns validates whether the arguments returned by the lambdaHabdler are valid or not. A valid lambdaHandler
```

Comment:
"lambdaHandler" instead of "lambdaHabdler"

Reply:
Updated